server/async_tcp_darwin.go

```diff
@@ -0,0 +1,166 @@
+/**
+Author: Yakshit Jain
+Date: 19 Oct 2022
+
+Implementation Reference:
+i) https://overengineered.dev/writing-an-event-loop-in-go
+ii) https://www.freebsd.org/cgi/man.cgi?query=kqueue&sektion=2
+**/
```

Comment:
Keep the references but remove the top header comments. The first line should be `package` you can move the reference just for the RunAsyncTCPServer function.

Reply:
We will define a common header for Dice DB and author information once the project reaches a certain milestone. The entire flow will be automated.

Reply:
Made the required changes.